### PR TITLE
FOX-218: Timer util function

### PR DESCRIPTION
**Pull request by ShiroKatake**
**Test scene (if any):**
- `Scenes/Brians Level Idea.unity`

## Changes summary:
- Extract timer util function

## Checklist before requesting a review:
- [x] I have run the game locally
- [x] I have performed self-review on my code
- [x] I have confirmed critical features still function

### CODE REVIEW

File: Assets/Scripts/Utils/TimeFormatter.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public static class TimeFormatter
+{
+    public static string Milliseconds(float timeMillisecond)
+    {
+        int minutes = (int)timeMillisecond / 60;
+        int seconds = (int)timeMillisecond % 60;
+        int milliseconds = (int)(timeMillisecond * 100) % 100;
+
+        var timeText = string.Format("{0:00}:{1:00}.{2:00}", minutes, seconds, milliseconds);
```
Comment (jess-rubino) [inline Assets/Scripts/Utils/TimeFormatter.cs:9]: are we doing 2:000 or 2:00?

Comment (ShiroKatake) [reply Assets/Scripts/Utils/TimeFormatter.cs]: good catch! We are doing 3 digits

File: Assets/Scripts/Utils/TimeFormatter.cs
Comment (sachdawg) [inline Assets/Scripts/Utils/TimeFormatter.cs:1]: the timer has an extra 0 in game at start of the milliseconds.

I think could fix with line 7 if its like 
int milliseconds = (int)(timeMillisecond * 1000) % 1000;

**Review by sachdawg — APPROVED** [review]
LGTM 🥳

File: Assets/Scripts/Utils/TimeFormatter.cs
Comment (ShiroKatake) [reply Assets/Scripts/Utils/TimeFormatter.cs:1]: noted, thank you

**Comment by ShiroKatake** [comment]
Link to build: https://github.com/WeAreBrian/project-fox-poc/actions/runs/6068644688

**Pull request merged**